civil/civil.go

```diff
@@ -107,6 +107,11 @@ func (d1 Date) After(d2 Date) bool {
 	return d2.Before(d1)
 }
 
+// IsZero reports whether date fields are empty.
```

Comment:
nit: instead of saying empty for all of these could we say something like ".. are set to their default value"

Reply:
okay. Done